go/enclave/enclave.go

```diff
@@ -337,44 +337,44 @@ func describeBSR(response *common.BlockSubmissionResponse) string {
 
 func (e *enclaveImpl) SubmitTx(tx common.EncryptedTx) responses.RawTx {
 	if atomic.LoadInt32(e.stopInterrupt) == 1 {
-		return responses.AsEmptyResponse()
+		return e.handlePlainTextError(errutil.NewSystemErr(fmt.Errorf("requested SubmitTx with the enclave stopping")))
```

Comment:
this should be:

`return nil, errutil.NewSystemErr(fmt.Errorf("requested SubmitTx with the enclave stopping")`